File: src/main/java/catmoe/fallencrystal/akanelimbo/util/menu/GUIEnchantUtil.java
package catmoe.fallencrystal.akanelimbo.util.menu;

public class GUIEnchantUtil {
    private GUIEnchantsList enchantment;
    private int level;

    public GUIEnchantUtil(GUIEnchantsList enchantment) {
        this(enchantment, 1);
    }

    public GUIEnchantUtil(GUIEnchantsList enchantment, int level) {
        this.enchantment = enchantment;
        this.level = level;
    }

    public GUIEnchantsList getEnchantment() {
        return enchantment;
    }

    public int getLevel() {
        return level;
    }

    public String toString() {
        return "d: \"" + enchantment.getString() + "\", lvl: " + level + "s";
    }
}

File: src/main/java/catmoe/fallencrystal/akanelimbo/util/menu/GUIEnchantUtil.kt
package catmoe.fallencrystal.akanelimbo.util.menu

class GUIEnchantUtil @JvmOverloads constructor(@JvmField val enchantment: GUIEnchantsList, @JvmField val level: Int = 1) {

    override fun toString(): String {
        return "d: \"" + enchantment.string + "\", lvl: " + level + "s"
    }
}